src/Main.ts

```diff
@@ -1211,6 +1212,28 @@ export class Main {
             });
         }
         log.info(`Set new access token for ${userId} (team: ${teamId}, puppeting: ${puppeting})`);
+        if (!existingTeam && !puppeting && this.teamSyncer) {
+            log.info("This is a new team, so syncing members");
+            try {
+                await this.teamSyncer.syncItems(
+                    teamId,
+                    await this.clientFactory.getTeamClient(teamId),
+                    "user",
+                );
+            } catch (ex) {
+                log.info("Failed to sync members", ex);
+            }
+
+            try {
+                await this.teamSyncer.syncItems(
+                    teamId,
+                    await this.clientFactory.getTeamClient(teamId),
+                    "channel",
+                );
+            } catch (ex) {
+                log.info("Failed to sync channels", ex);
```

Comment:
I think this should be a warning.
```suggestion
                log.warn("Failed to sync members", ex);
            }

            try {
                await this.teamSyncer.syncItems(
                    teamId,
                    await this.clientFactory.getTeamClient(teamId),
                    "channel",
                );
            } catch (ex) {
                log.warn("Failed to sync channels", ex);
```